cfme/tests/control/test_basic.py

```diff
@@ -640,6 +671,9 @@ def test_alert_profile_crud(request, appliance, alert_profile_class):
         casecomponent: Control
         caseimportance: critical
         initialEstimate: 1/12h
+
+    Bugzilla:
+        1723815
```

Comment:
Looks good , please also add 
@pytest.mark.meta(automates=[1723815]) to the test

Reply:
@sshveta it is there already :smiley:

Reply:
:+1:  Yeah just saw :)